VideoLocker/src/main/java/org/edx/mobile/view/CourseUnitNavigationActivity.java

```diff
@@ -156,16 +166,53 @@ private void setCurrentUnit(CourseComponent component){
 
     private void tryToUpdateForEndOfSequential(){
         int curIndex = pager.getCurrentItem();
-        setCurrentUnit( pagerAdapter.getUnit(curIndex) );
+        setCurrentUnit(pagerAdapter.getUnit(curIndex));
 
-        View prevButton = findViewById(R.id.goto_prev);
-        Button nextButton = (Button) findViewById(R.id.goto_next);
-        prevButton.setEnabled(curIndex > 0);
-        nextButton.setEnabled(curIndex < pagerAdapter.getCount() - 1);
+        mPreviousBtn.setEnabled(curIndex > 0);
+        mNextBtn.setEnabled(curIndex < pagerAdapter.getCount() - 1);
  
         findViewById(R.id.course_unit_nav_bar).requestLayout();
 
         setTitle(selectedUnit.getDisplayName());
+
+        // Khalid: code below decides to show Next/Previous Unit name or only Next/Previous
```

Comment:
Don't need your name on the comment since it's already in the git history. Linking to a ticket can be useful though.